caffeine/src/main/scala/akka/caffeine/Aggregator.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.caffeine
+
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * Aggregation contract
```

Comment:
OK, an `Aggregator` does aggregation. I would have concluded that from the class name. Could you please provide a more descriptive comment here?